build.gradle

```diff
@@ -36,6 +36,10 @@ subprojects {
 
     apply plugin: 'com.github.johnrengelman.shadow'
 
+    shadowJar {
+        zip64 true // Required to allow more than 65535 entries in final archive
```

Comment:
The ffmpeg-platform artifact contains the binaries required for all platforms allowing the jar to be portable once built.
Unfortunately, with all available platform binaries the total number of files in the final fat jar [exceeds 65535](https://en.wikipedia.org/wiki/ZIP_(file_format)#ZIP64) and therefore the jar must be compressed as a zip64 rather than a regular zip file.

Reply:
Makes sense, thanks!